proto/sifnode/tokenregistry/v1/types.proto

```diff
@@ -21,20 +21,19 @@ enum Permission {
 }
 
 message RegistryEntry {
-  bool is_whitelisted = 1;
```

Comment:
Has this flag ever been stored in the production tokenRegistry? If so, we cannot remove it, only rename it.

Reply:
Looking at develop branch the registry entry looks like
```proto
message RegistryEntry {
  reserved 1;
  int64 decimals = 2;
  string denom = 3;
  string base_denom = 4;
  string path = 5;
  string ibc_channel_id = 6;
  string ibc_counterparty_channel_id = 7;
  string display_name = 8;
  string display_symbol = 9;
  string network = 10;
  string address = 11;
  string external_symbol = 12;
  string transfer_limit = 13;
  repeated Permission permissions = 15;
  ```
so we cant just delete this entry can we?

Reply:
I am not sure if it is used in the production. I checked the master branch, the is_whitelisted also removed and  set it as reserved. We have different approached to handle it, via data migration, or add v2 for our proto buf. For now, I will follow the develop and master branch, set it as reserved.